src/System.IO.FileSystem/tests/WinRT_BrokeredFunctions.cs

```diff
@@ -0,0 +1,207 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.InteropServices;
+using Xunit;
+using Windows.Storage;
+
+namespace System.IO.Tests
+{
+    [PlatformSpecific(TestPlatforms.Windows)]
+    [SkipOnTargetFramework(~(TargetFrameworkMonikers.Uap | TargetFrameworkMonikers.UapAot))]
+    public partial class WinRT_BrokeredFunctions : FileSystemTest
+    {
+        private static string s_musicFolder = StorageLibrary.GetLibraryAsync(KnownLibraryId.Music).AsTask().Result.SaveFolder.Path;
+
+        [Fact]
+        public void CopyFile_ToBrokeredLocation()
+        {
+            string testFile = GetTestFilePath();
+            File.Create(testFile).Dispose();
+
+            string destination = Path.Combine(s_musicFolder, "CopyToBrokeredLocation_" + Path.GetRandomFileName());
+            try
+            {
+                Assert.False(File.Exists(destination), "destination shouldn't exist before copying");
+                File.Copy(testFile, destination);
+                Assert.True(File.Exists(testFile), "testFile should exist after copying");
+                Assert.True(File.Exists(destination), "destination should exist after copying");
+            }
+            finally
+            {
+                File.Delete(testFile);
+                File.Delete(destination);
+            }
+        }
+
+        [Fact]
+        public void CreateDirectory()
+        {
+            string testFolder = Path.Combine(s_musicFolder, "CreateDirectory_" + Path.GetRandomFileName());
+            try
+            {
+                Assert.False(Directory.Exists(testFolder), "destination shouldn't exist");
+                Directory.CreateDirectory(testFolder);
+                Assert.True(Directory.Exists(testFolder), "destination should exist");
+            }
+            finally
+            {
+                Directory.Delete(testFolder);
+            }
+        }
+
+        [Fact]
+        public void DeleteFile()
+        {
+            string testFile = Path.Combine(s_musicFolder, "DeleteFile_" + Path.GetRandomFileName());
+            CreateFileInBrokeredLocation(testFile);
+            Assert.True(File.Exists(testFile), "testFile should exist before deleting");
+            File.Delete(testFile);
+            Assert.False(File.Exists(testFile), "testFile shouldn't exist after deleting");
+        }
+
+        [Fact]
+        public void FindFirstFile()
+        {
+            string subFolder = Path.Combine(s_musicFolder, "FindFirstFile_SubFolder_" + Path.GetRandomFileName());
+            Directory.CreateDirectory(subFolder);
+            string testFile = Path.Combine(subFolder, "FindFirstFile_SubFile_" + Path.GetRandomFileName());
+            CreateFileInBrokeredLocation(testFile);
```

Comment:
These two lines should probably be inside the try so that if they happen to throw, the finally will still delete the directory we created in the music folder.

Also, in case something does go wrong and files get left behind, it'd be good to name any files/folders we put into a non-tmp location with a name that clearly associates it with corefx tests, e.g. "corefx_tests_FindFirstFile_SubFolder...", etc.  If the test process is killed at the wrong time, however, and I find myself looking at a bunch of folders, I'd like to be able to easily identify the ones that are from corefx tests and that are thus safe to delete.